src/main/java/racingcargame/domain/RacingCarGame.java

```diff
@@ -0,0 +1,30 @@
+package racingcargame.domain;
+
+import java.util.List;
+
+public class RacingCarGame {
+
+	public static final int RANDOM_NUMBER_MIN_BOUNDARY = 0;
+	public static final int RANDOM_NUMBER_MAX_BOUNDARY = 9;
+
+	private Cars racingCars;
+
+	public RacingCarGame() {
+	}
+
+	public void init(String carNames) {
+		this.racingCars = new Cars(carNames);
+	}
+
+	public Cars getRacingCars() {
+		return racingCars;
+	}
+
+	public void gameStart() {
+		racingCars.tryMove(RANDOM_NUMBER_MIN_BOUNDARY, RANDOM_NUMBER_MAX_BOUNDARY);
+	}
+
+	public List<String> result() {
```

Comment:
메소드 이름이 명사 하나만 있네요.
동사를 추가하는게 좋을 것 같아요.

```suggestion
	public List<String> getResult() {
```

Reply:
메서드 네이밍에 대해 일관성있는 규칙을 가지도록 리팩터링 하겠습니다.